cds_ils/importer/providers/cds/rules/base.py

```diff
@@ -128,9 +128,12 @@ def tags(self, key, value):
 def special_serials(self, key, value):
     """Translates serial fields."""
     _migration = self["_migration"]
-    _serials = _migration["serials"]
+    _serials = _migration.get("serials", False)
```

Comment:
`serials` is an empty list by default not `False`

Reply:
This is on purpose, this is used by journals too and they don't have `serials` in the `_migration`, so if this is the case we set it to false and in the if statemente below it will translate the `document_type`.

Reply:
in the if statement below will work the same for the empty list, as follows:
```
if not result_a or not _serials:
```
while in the line 138, there is usage of `_serials` which assumes it is a list, and it will fail if `_serials` is False